runtime-builder/build.sh

```diff
@@ -14,18 +14,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-usage() { echo "Usage: $0 [project_id] [go_version]"; exit 1; }
+usage() { echo "Usage: $0 GO_VERSION [DEBIAN_TAG]"; exit 1; }
```

Comment:
[GO_VERSION] for consistency.  However, I generally use <arg> to indicate arguments instead, so perhaps <go_version> <debian_tag>.